docs/molgenis/run_systemd.md

```diff
@@ -0,0 +1,179 @@
+
+# Install Molgenis emx2 as a system service 
+We Support redhat and ubuntu based installations.
+
+
+## Java 
+
+Molgenis-emx2 runs on java > 17
```

Comment:
idem voor MOLGENIS EMX2. 
Daarnaast staat in de installation guide: Install java (we use adopt [OpenJDK 17](https://adoptium.net/)). Ik draai ook op 17.